DeleteMiddleNode.py

```diff
@@ -0,0 +1,36 @@
+class ListNode:
+    def __init__(self, value=0, next=None):
+        self.value = value
+        self.next = next
+
+def deleteNode(node):
+    if node is None or node.next is None:
+        return
+    nextNode = node.next
+    node.value = nextNode.value
+    node.next = nextNode.next
+    del nextNode
+
+def printList(current):
```

Comment:
`current` is really not a great variable name because it's a placeholder and doesn't really mean anything.